File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Openf1CarData SDK exists test

require_once __DIR__ . '/../openf1cardata_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = Openf1CarDataSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# Openf1CarData SDK exists test

require "minitest/autorun"
require_relative "../Openf1CarData_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = Openf1CarDataSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
